jinahub/rankers/DPRReaderRanker/README.md

```diff
@@ -2,8 +2,76 @@
 
  **DPRReaderRanker** performs a re-ranking of the matches for each document (question), as well as the answer spans extraction for each match. It uses the DPR Reader model to re-rank documents based on cross-attention between the question and the potential answer passages.
 
-The **DPR** model was originally proposed in [Dense Passage Retrieval for Open-Domain Question Answering](https://arxiv.org/abs/2004.04906).
+The **DPR** model was originally proposed in [Dense Passage Retrieval for Open-Domain Question Answering](https://arxiv.org/abs/2004.04906). This ranker can be used together with **[DPRTextEncoder](https://hub.jina.ai/executor/awl0jxog)** to use the entire
+DPR pipeline for QA tasks.
 
+## Usage
```

Comment:
As I understand from latest comments, we do not want to have so much info in the README.

Reply:
I thought that refers only to things that could be "auto-generated". For the highlighted executors we were actually adding such more informative examples.